tools/osbuilder/Makefile

```diff
@@ -220,9 +222,13 @@ install-scripts:
 
 .PHONY: clean
 clean:
-	rm -rf $(TARGET_ROOTFS_MARKER) $(TARGET_ROOTFS) $(TARGET_IMAGE) $(TARGET_INITRD) $(DRACUT_OVERLAY_DIR) $(TARGET_IGVM) $(TARGET_IGVM_DEBUG) $(TARGET_IGVM_MSMT) $(TARGET_IGVM_DEBUG_MSMT) $(TARGET_IGVM_LOG)
+	rm -rf $(TARGET_ROOTFS_MARKER) $(TARGET_ROOTFS) $(IMAGES_BUILD_DEST)/*.img $(TARGET_INITRD) $(DRACUT_OVERLAY_DIR) $(TARGET_IGVM) $(TARGET_IGVM_DEBUG) $(TARGET_IGVM_MSMT) $(TARGET_IGVM_DEBUG_MSMT) $(TARGET_IGVM_LOG)
```

Comment:
Noticed small bug when supporting builds of singular debug/release uvm: make clean only tries to clean up the default kata-containers.img. Should instead clean all img files in the IMAGES_BUILD_DEST.